.buildkite/scripts/build_component.sh

```diff
@@ -29,16 +44,28 @@ set_hab_binary() {
         # runs that may take place on it.
         sudo hab pkg install "${hab_ident}"
         sudo hab pkg install "$(buildkite-agent meta-data get studio-version)"
-        declare -g hab_binary="/hab/pkgs/${hab_ident}/bin/hab"
+        hab_binary="/hab/pkgs/${hab_ident}/bin/hab"
         declare -g new_studio=1
     else
-        echo "Buildkite metadata NOT found; using previously-installed hab binary"
-        hab_binary="$(which hab)"
-        declare -g hab_binary
+        echo "Buildkite metadata NOT found; using previously-installed hab binary: $hab_binary"
     fi
+    declare -g hab_binary
     echo "--- :habicat: Using $(${hab_binary} --version)"
 }
 
+install_hab_kernel2_binary() {
+    local hab_src_url tempdir
+    hab_src_url="http://habitat-boostrap-artifacts.s3.amazonaws.com/x86_64-linux-kernel2/stage2/habitat-stage2-x86_64-linux-kernel2-latest"
+    tempdir=$(mktemp -d hab-kernel2-XXXX)
+
+    pushd $tmpdir >/dev/null
+    wget "$hab_src_url" hab-x86_64-linux-kernel2
```

Comment:
We use `curl` everywhere else in our pipeline scripts. For consistency, it'd be good to pick either `curl` or `wget` and just use that everywhere.

I don't have particularly strong opinions either way; it's `curl` everywhere only because that's what I personally use the most, and I wrote the rest of the scripts 😄

Reply:
Copy/paste 😄   Will update to curl